Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfilesInspector.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Inspectors.Profiles
+{
+    [CustomEditor(typeof(MixedRealityControllerMappingProfiles))]
+    public class MixedRealityControllerMappingProfilesInspector : MixedRealityBaseConfigurationProfileInspector
+    {
+        private SerializedProperty mouseControllerMappingProfile;
+        private SerializedProperty touchScreenControllerMappingProfile;
+        private SerializedProperty windowsMixedRealityControllerMappingProfile;
+        private SerializedProperty viveWandControllerMappingProfile;
+        private SerializedProperty oculusTouchControllerMappingProfile;
+        private SerializedProperty oculusRemoteControllerMappingProfile;
+        private SerializedProperty genericUnityControllerMappingProfile;
+        private SerializedProperty genericOpenVRControllerMappingProfile;
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            if (!CheckMixedRealityConfigured(false))
+            {
+                return;
+            }
+
+            if (!MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled ||
+                 MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.InputActionsProfile == null)
+            {
+                return;
+            }
+
+            mouseControllerMappingProfile = serializedObject.FindProperty("mouseControllerMappingProfile");
```

Comment:
Need to shorten the displayed names, as they are cut off in the inspector.
Or possibly switch the profile line item to a generic image for the platform maybe?
(I really liked the graphical view, which is now hidden behind these line items)

Reply:
Agreed. I wasn't quite sure what to do there for that, as we need to "select" the scriptable object profile to get it to display. I'll see what I can do here.